getData.js

```diff
@@ -0,0 +1,46 @@
+const apiUrl = "http://localhost:3000/api";
+
+const getData = (app, command, id = "") => {
+  return fetch(`${apiUrl}/${app}/${command}/${id}`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      return data;
+    });
+};
+
+const getBotToken = () => {
+  return getData("botSetting", "getBotToken");
+};
+
+const getHelloMessage = () => {
+  return getData("languageProfile", "getHelloMessage", "1");
+};
+
+const getLanguages = () => { 
+    return getData("languageProfile", "all")
+}
+const getAllData = () => {
```

Comment:
Try to omit "Data" in function, class or variable names since it adds no real value to readability. getAppSettings or getBotSettings would be way better here as it better explains the purpose of the function